### ISSUE TITLE
Suggestion for modify win32.js related drive letter's issue

### ISSUE THREAD

**Issue by Mon4455** [iximiuz/node-diskusage-ng#6]
Hello!
I suggest fixing the function diskname() defined in "win32.js" as follows
because stdout of WMIC command always contains Upper case drive letter.
If the argument string of diskname() contains lower case drive letter, diskusage() will be failed.

Before:
```
function diskname(path) {
    var parsed = Path.parse(Path.normalize(path));
    var disk = parsed.root.split('\\')[0];
    if (!isDiskname(disk)) {
        throw new Error('Bad path');
    }
    return disk;
}
```
After:
```
function diskname(path) {
    var parsed = Path.parse(Path.normalize(path));
    var disk = parsed.root.split('\\')[0].toUpperCase();
    if (!isDiskname(disk)) {
        throw new Error('Bad path');
    }
    return disk;
}
```

**Comment by iximiuz** [comment]
Hi @Mon4455! Thanks for opening this issue! I'll take a closer look as soon as I can!

**Comment by iximiuz** [comment]
Hey hey! Just released a new version - [v1.0.2](https://www.npmjs.com/package/diskusage-ng). Thanks for the suggestion!
